statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -1716,17 +1720,24 @@ def test_inst_causality(self, causing, signif=0.05):
         -------
         results : dict
             A dict holding the test's results. The dict's keys are:
-            * "statistic" : float
-                The claculated test statistic.
-            * "crit_value" : float
-                The critical value of the \Chi^2-distribution.
-            * "pvalue" : float
-                The p-value corresponding to the test statistic.
-            * "df" : float
-                The degrees of freedom of the \Chi^2-distribution.
-            * "conclusion" : str {"reject", "fail to reject"}
-                 Whether H0 can be rejected or not.
-            * "signif" : float
+
+            "statistic" : float
+              The calculated test statistic.
+
+            "crit_value" : float
+              The critical value of the Chi^2-distribution.
+
+            "pvalue" : float
+              The p-value corresponding to the test statistic.
+
+            "df" : float
+              The degrees of freedom of the Chi^2-distribution.
+
+            "conclusion" : str {"reject", "fail to reject"}
+              Whether H0 can be rejected or not.
+
+            "signif" : float
+              Signifigance level
```

Comment:
was this necessary
I thought the list with would work as rst

Reply:
Signifigance --> Significance